rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -44,6 +44,11 @@ THE SOFTWARE.
 #endif
 #include "randombboxcrop_meta_data_reader.h"
 #define MAX_STRING_LENGTH 100
+#define MAX_OBJECTS 50 // Max number of objects/image in COCO dataset is 93 
```

Comment:
Why are we setting max objects as 50 if the max bboxes is 93 in COCO dataset?

Reply:
An arbitrary number, basically the average